microsoft.go

```diff
@@ -155,12 +159,25 @@ func (b MicrosoftBlobBackend) PutObject(path string, content []byte) error {
 
 	err := blobReference.PutAppendBlob(nil)
 	if err == nil {
-		err = blobReference.AppendBlock(content, nil)
+		err = writeToBlob(content, blobReference)
 	}
 
 	return err
 }
 
+func writeToBlob(content []byte, blobRef *microsoft_storage.Blob) error {
+	for offset := 0; offset < len(context); offset += maxChunkSize {
```

Comment:
thanks for updating the var name @yuanjumao, we'll need to update it everywhere its used in this function as well!